apps/web/components/LatestNewsSection/LatestNewsSection.tsx

```diff
@@ -23,8 +26,45 @@ const LatestNewsSection: React.FC<LatestNewsProps> = ({
   label,
 }) => {
   const newsItems = items.slice(0, 3)
+  const { width } = useWindowSize()
+  const [isMobile, setIsMobile] = useState(false)
 
-  return (
+  useIsomorphicLayoutEffect(() => {
+    if (width < theme.breakpoints.md) {
+      return setIsMobile(true)
+    }
+    setIsMobile(false)
+  }, [width])
+
+  const renderMobile = () => (
+    <GridContainer>
+      <GridRow>
+        <GridColumn span="12/12">
```

Comment:
would it make sense to drop <GridRow> and <GridColumn> here (if the components span 100% by default)?

Reply:
Do you mean to add that as a feature to Grid? Sounds sensible.